src/connection.rs

```diff
@@ -5,24 +5,9 @@ use std::rc::Rc;
 
 use mio::tcp::TcpStream;
 use mio::Timeout as TimeoutHandle;
-use mio::{
```

Comment:
Takes a lot of space for something you barely ever read and which tooling manages for you (ideally). I'm happy having it collapsed.